omnigibson/simulator.py

```diff
@@ -118,18 +129,32 @@ def _launch_app():
     # Copy the OmniGibson kit file to the Isaac Sim apps directory. This is necessary because the Isaac Sim app
     # expects the extensions to be reachable in the parent directory of the kit file. We copy on every launch to
     # ensure that the kit file is always up to date.
-    assert "EXP_PATH" in os.environ, "The EXP_PATH variable is not set. Are you in an Isaac Sim installed environment?"
+
+    # check if EXP_PATH exists, if not proceed with pip install path
     kit_file = Path(__file__).parent / kit_file_name
-    kit_file_target = Path(os.environ["EXP_PATH"]) / kit_file_name
+    try:
+        kit_file_target = Path(os.environ["EXP_PATH"]) / kit_file_name
+    except KeyError:
+        kit_file_target = os.path.join(site.getsitepackages()[0], "isaacsim", "apps", kit_file_name)
+    except:
+        # if neither of these paths exist, raise error to user
+        raise ValueError(
+            f"Could not find the Isaac sim file file path. Please set this variable to the path of your Isaac Sim installation."
+        )
```

Comment:
if "EXP_PATH" in os.environ:
    ...
else:
    ...

Reply:
Find out a more principled way for getting the isaacsim package's path (e.g. you can try importing isaacsim and then doing isaacsim.__package__ or maybe isaacsim.__file__?)

import isaacsim
exp_path = os.path.join(os.path.dirname(isaacsim.__file__), "isaacsim", "apps", kit_file_name)